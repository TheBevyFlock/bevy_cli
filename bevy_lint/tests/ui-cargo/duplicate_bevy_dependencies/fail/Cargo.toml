[package]
name = "multiple-bevy-versions"
version = "0.1.0"
publish = false
edition = "2021"

[workspace]

[dependencies]
bevy = { version = "0.14.2" }
bevy_ecs_ldtk = "0.5.0"
