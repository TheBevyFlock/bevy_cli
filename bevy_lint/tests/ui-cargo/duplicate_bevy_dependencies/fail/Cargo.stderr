error: Mismatching versions of `bevy` found
  --> Cargo.toml:11:26
   |
11 | leafwing-input-manager = "0.13"
   |                          ^^^^^^
   |
help: Expected all crates to use `bevy` 0.14.2
  --> Cargo.toml:10:8
   |
10 | bevy = { version = "0.14.2" }
   |        ^^^^^^^^^^^^^^^^^^^^^^
   = note: `#[deny(bevy::duplicate_bevy_dependencies)]` on by default

error: could not compile `multiple-bevy-versions` (bin "multiple-bevy-versions") due to 1 previous error
Check failed: exit status: 101.
